src/constants.js

```diff
@@ -1,5 +1,5 @@
 export const INLINE = {};
 export const RUNTIME = {};
-export const BUNDLED = {};
+export const EXTERNAL = {};
 
-export const HELPERS = '\0babelHelpers';
+export const HELPERS = 'rollupPluginBabelHelpers';
```

Comment:
I assume the `\0` was here make it less likely for the name to conflict with a real imports. Seems like we should keep that?

Reply:
it didn't work with `\0` in place, aint sure why - didn't bother to investigate it further, gonna check in few moments if I can find out why I've encountered problem with that

Reply:
If we can't figure it out, that's alright. Not gonna block on this.

Reply:
`acorn` is throwing with `SyntaxError: Unexpected character '' (6:12)`